app/models/user.rb

```diff
@@ -17,4 +17,6 @@ class User < ApplicationRecord
   has_many :reviews, class_name: "Review", foreign_key: "student_id", dependent: :destroy
 
   accepts_nested_attributes_for :attendances, :group_member, :student_information
+
+  scope :my_students, ->(students_ids) { where("id IN (?)", students_ids) }
```

Comment:
move this to a class method where you send the students_ids array and try using `where(id: students_ids)` to see if it works pls

Reply:
You're right! works correctly.